src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -109,7 +109,7 @@ public static IEnumerable<object[]> GetAsync_AllowedSSLVersion_Succeeds_MemberDa
 #endif
         }
 
-        [Theory]
+        [ConditionalTheory]
```

Comment:
Should this have the condition as the argument?

Reply:
or does this just cause that SkipTestException is handled differently?

Reply:
yes, This is so throw SkipTestException causes skip, not failure.